src/Mod/Assembly/App/AssemblyObject.h

```diff
@@ -57,6 +62,8 @@ namespace Assembly
 class JointGroup;
 class ViewGroup;
 
+using ObjRefPair = std::pair<App::DocumentObject*, App::PropertyXLinkSub*>;
```

Comment:
I don't love the use of std::pair here (it makes later code quite opaque, as the reader has to remember what `.first` and `.second` refer to. This seems to get used often enough to justify being a struct, in my opinion.

Reply:
Sounds reasonable.

Reply:
Done